wemix/admin.go

```diff
@@ -777,6 +554,7 @@ func StartAdmin(stack *node.Node, datadir string) {
 	if err != nil {
 		return
 	}
+	admin.contracts, _ = admin.getRegGovEnvContracts(nil, nil)
```

Comment:
We need to check error.

Reply:
This code was removed as unnecessary because it runs again in `wemixAdmin.run()`, `wemixAdmin.update()`